daemon/logger/etwlogs/etwlogs_windows.go

```diff
@@ -73,10 +63,20 @@ func New(info logger.Info) (logger.Logger, error) {
 
 // Log logs the message to the ETW stream.
 func (etwLogger *etwLogs) Log(msg *logger.Message) error {
-	// TODO(thaJeztah): log structured events instead and use provider.WriteEvent().
-	m := createLogMessage(etwLogger, msg)
+	err := provider.WriteEvent("ContainerLogs", []etw.EventOpt{etw.WithLevel(0)}, etw.WithFields(
```

Comment:
FWIW; not sure how (and where) the first field (`ContainerLogs`) shows up, and if it's important.


As to this PR: The third alternative could be to keep the old code (for one release?) and send logs _both_ as structured and the old string format.

I'm not familiar enough with working with these logs to tell if that would be more (or _less_) disruptive.